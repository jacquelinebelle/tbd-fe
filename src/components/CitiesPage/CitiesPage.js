import React, { Component } from 'react'
export class CitiesPage extends Component {

  componentDidMount () {
      const script = document.createElement("script");
      script.src = "https://teleport.org/assets/firefly/widget-snippet.min.js";
      script.async = true;
      document.body.appendChild(script);
  }

  render() {
    return (
    <div>
        <a class="teleport-widget-link" href="https://teleport.org/cities/belize-city/">Life quality score - Belize City</a><script async class="teleport-widget-script" data-url="https://teleport.org/cities/belize-city/widget/scores/?currency=USD" data-max-width="770" data-height="977" src="https://teleport.org/assets/firefly/widget-snippet.min.js"></script>
    </div>
    )
  }
}

export default CitiesPage

