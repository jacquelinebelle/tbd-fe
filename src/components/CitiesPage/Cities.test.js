import React from "react";
import { shallow } from "enzyme";
import { CitiesPage } from "./CitiesPage";

describe('Cities Page', () => {
let wrapper;
  beforeEach(() => {
    wrapper = shallow(<CitiesPage />)
  })
  it("SHould match the snapshot", () => {
    expect(wrapper).toMatchSnapshot()
  })
})
