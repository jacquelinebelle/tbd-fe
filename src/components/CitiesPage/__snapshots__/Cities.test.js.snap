// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cities Page SHould match the snapshot 1`] = `
<div>
  <a
    class="teleport-widget-link"
    href="https://teleport.org/cities/belize-city/"
  >
    Life quality score - Belize City
  </a>
  <script
    async={true}
    class="teleport-widget-script"
    data-height="977"
    data-max-width="770"
    data-url="https://teleport.org/cities/belize-city/widget/scores/?currency=USD"
    src="https://teleport.org/assets/firefly/widget-snippet.min.js"
  />
</div>
`;
